packages/vee-validate/src/types/forms.ts

```diff
@@ -4,11 +4,16 @@ import { FieldValidationMetaInfo } from '../../../shared';
 import { Path, PathValue } from './paths';
 import { PartialDeep } from 'type-fest';
 
-export interface ValidationResult {
+export interface ValidationResult<TValue> {
```

Comment:
This is a public interface, lets default the generic `TValue` to `unknown`.

Reply:
Fixed